libmemcached-1.0/struct/memcached.h

```diff
@@ -124,6 +124,7 @@ struct memcached_st {
     uint32_t max_pool_size;
     int32_t version; // This is used by pool and others to determine if the memcached_st is out of date.
     struct memcached_array_st *filename;
+    struct memcached_array_st *source_address;
```

Comment:
I don't know what a `memcached_array_st` is and I'm pretty certain I don't want to know either.

Maybe an array of characters that looks a little bit like a string, but is _specific_ to memcached in some way. Maybe the string is terminated with `6d 65 6d 63 61 63 68 65 64`.